.travis.yml

```diff
@@ -5,7 +5,7 @@ git:
 
 sudo: required
 
-dist: trusty
```

Comment:
![image](https://user-images.githubusercontent.com/28776356/72488527-2c4af000-384c-11ea-85ee-30174da12fe3.png)
When using ubuntu 14.04(trusty), the matching docker version is 17.9. This error is reported when starting the pod/container, but the status of the Pod is Running.
So the previous e2e runs have problems, now upgrade to 16.04, the pod is running normally.